src/riak_kv_pb_timeseries.erl

```diff
@@ -487,24 +487,24 @@ missing_helper_module(Table, BucketProps) when is_binary(Table), is_list(BucketP
     end.
 
 %%
--spec missing_type_response(BucketType::binary()) -> #rpberrorresp{}.
-missing_type_response(BucketType) ->
+-spec missing_type_response(Table::binary()) -> #rpberrorresp{}.
+missing_type_response(Table) ->
     make_rpberrresp(
         ?E_MISSING_TYPE,
-        flat_format("Bucket type ~s is missing.", [BucketType])).
+        flat_format("Time Series table ~s is missing.", [Table])).
```

Comment:
I'd say `s/is missing/does not exist/'  "is missing" sounds like it was there, but now it's not.